packages/app-content-pages/src/screens/About/components/OurMission.js

```diff
@@ -1,14 +1,52 @@
-import { Anchor, Heading, Paragraph } from 'grommet'
+import {
+  Anchor,
+  Box,
+  Button,
+  Heading,
+  Image,
+  Paragraph,
+  Text,
+  Tip
+} from 'grommet'
+import { CircleInformation } from 'grommet-icons'
 import { Trans, useTranslation } from 'next-i18next'
-import Link from 'next/link'
+import styled from 'styled-components'
 
-function LinkInParagraph({ children, href }) {
-  return (
-    <Anchor as={Link} href={href}>
-      {children}
-    </Anchor>
-  )
-}
+import Stats from './Stats/Stats.js'
+
+const VideoWrapper = styled(Box)`
+  position: relative;
+  padding-bottom: 56.25%; // Creates a responsive 9:16 div
```

Comment:
Could css's `aspect-ratio` be used here?

https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio

Reply:
Yes! Added 👍